pkg/occlient/occlient_test.go

```diff
@@ -8,9 +8,45 @@ import (
 	"testing"
 
 	routev1 "github.com/openshift/api/route/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	ktesting "k8s.io/client-go/testing"
 )
 
+func TestDeletePVC(t *testing.T) {
+	tests := []struct {
+		name    string
+		pvcname string
+		wantErr bool
+	}{
+		{
+			name:    "storage 10Gi",
+			pvcname: "postgresql",
+			wantErr: false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			fakeClient, fakeClientSet := FakeNew()
+
+			fakeClientSet.Kubernetes.PrependReactor("delete", "persistentvolumeclaims", func(action ktesting.Action) (bool, runtime.Object, error) {
+				return true, nil, nil
+			})
+
+			err := fakeClient.DeletePVC(tt.name)
+
+			//Checks for error in positive cases
+			if !tt.wantErr == (err != nil) {
+				t.Errorf(" client.DeletePVC(name) unexpected error %v, wantErr %v", err, tt.wantErr)
+			}
+
+			// Check for validating actions performed
+			if (len(fakeClientSet.Kubernetes.Actions()) != 1) && (tt.wantErr != true) {
+				t.Errorf("expected 1 action in DeletePVC got: %v", fakeClientSet.Kubernetes.Actions())
+			}
+		})
```

Comment:
Just check if `fakeClientSet.Kubernetes.Actions()[0].(ktesting.DeleteAction).GetName() returns name

Reply:
yep, it did return the `tt.name`,

added checks for validating it.

thanks, @ashetty1